src/rest-server/src/models/v2/job.js

```diff
@@ -142,7 +142,7 @@ const generateYarnContainerScript = (frameworkName, userName, config, frameworkD
     azRDMA: azureEnv.azRDMA === 'true' ? true : false,
     reqAzRDMA: false,
     inspectPidFormat: '{{.State.Pid}}',
-    inspectOOMKilledFormat: '{{.State.OOMKilled}}',
+    infoDefaultRuntimeFormat: '{{json .DefaultRuntime}}',
```

Comment:
Seems this field is static and we can put it into the shell script.

Reply:
the `{{ ... }}` will be replaced by `mustache` in shell script template, so we have to put it here as a workaround.